setup.py

```diff
@@ -3,6 +3,9 @@
 with open('README.md', 'r', encoding='utf-8') as f:
     long_description = f.read()
 
+with open('requirements/requirements.txt') as f:
+    requirements = f.readlines() + ['whist-core~=0.1.0rc2']
```

Comment:
First of all, thx for your PR. 
One thing, do we need the whist core here, as it's already in the requirements file?

Reply:
Good spot didn't check that one cloesely. Fixed.